subprojects/docs/src/docs/userguide/authoring-builds/writing_build_scripts.adoc

```diff
@@ -200,87 +211,119 @@ include::{snippetsPath}/tutorial/configureObjectUsingScript/tests/configureObjec
 ====
 
 [[groovy-dsl-basics]]
-== Some Groovy basics
-
-[TIP]
-Looking for some Kotlin basics, the link:https://kotlinlang.org/docs/reference/[Kotlin reference documentation] and link:https://kotlinlang.org/docs/tutorials/koans.html[Kotlin Koans] should be useful to you.
+== Helpful Groovy Syntax
 
-The https://docs.groovy-lang.org/latest/html/documentation/index.html[Groovy language] provides plenty of features for creating DSLs, and the Gradle build language takes advantage of these. Understanding how the build language works will help you when you write your build script, and in particular, when you start to write custom plugins and tasks.
+Groovy provides many features to help create DSLs. These features can help you write better build scripts, plugins, and tasks.
 
+[TIP]
```

Comment:
Why'd we drop the link to Groovy language docs but keep the Kotlin one?

Reply:
Added back, thank you!